packages/manager-tools/manager-generator/template/README.md

```diff
@@ -0,0 +1,211 @@
+# {{appNameKebab}} — OVHcloud Manager Application
+
+> Flavor: **{{routeFlavor}}**  
```

Comment:
IMO, Route flavor is more technical and does not seem like an information to be exposed from README file.